adoc/admin-disaster-recovery-velero.adoc

```diff
@@ -0,0 +1,707 @@
+mkdir -p bucket
+./minio server bucket &
+
+# Download Minio client
+wget https://dl.min.io/client/mc/release/linux-amd64/mc
+chmod +x mc
+
+# Setup Minio server
+./mc config host add velero http://localhost:9000 $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+
+# Create bucket on Minio server
+./mc mb -p velero/velero
+----
+
++
+For the rests of the S3-compatible storage providers, refer to https://velero.io/docs/v1.2.0/supported-providers/[Velero Supported Providers].
+
+. Volume Snapshotter
++
+[NOTE]
+The volume snapshotter snapshots the persistent volume if the volume supports the snapshot API to do volume snapshot. If the volume does not supports snapshot API or does not have supported providers, Velero leverages _restic_ to do persistent volume backup and restore.
+
+[IMPORTANT]
+Please be well-known the persistent volume in the supported volume snapshotter provides. Then adminstrators have to determine whether to deploy the _restic daemonset_ or not.
+
+.. Public Cloud Storage Providers
++
+|=======
+| Provider | Volume Snapshotter | Plugin Provider Repo
+| Amazon Web Services (AWS) | AWS EBS | https://github.com/vmware-tanzu/velero-plugin-for-aws[Velero plugin for AWS]
+|=======
+
+.. Community Supported Providers
++
+|=======
+| Provider | Volume Snapshotter | Plugin Documentation
+| Portworx | Portworx Volume | https://docs.portworx.com/scheduler/kubernetes/ark.html[	Portworx]
+| DigitalOcean | DigitalOcean Volumes Block Storage | https://github.com/StackPointCloud/ark-plugin-digitalocean[StackPointCloud]
+| OpenEBS | OpenEBS CStor Volume | https://github.com/openebs/velero-plugin[OpenEBS]
+| AlibabaCloud | AlibabaCloud | https://github.com/AliyunContainerService/velero-plugin[AlibabaCloud]
+| Hewlett Packard | HPE Storage | https://github.com/hpe-storage/velero-plugin[Hewlett Packard	Slack]
+|=======
+
+. Object Storage Credential
+
+.. Public Cloud Providers
+... AWS S3
++
+Create the credential file `credentials-velero` in the local machine.
++
+```
+[default]
+aws_access_key_id=<AWS_ACCESS_KEY_ID>
+aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
+```
++
+Please refer to https://github.com/vmware-tanzu/velero-plugin-for-aws/tree/v1.0.0[Velero Plugin For AWS].
+
+... Google Cloud Storage
++
+Create the credential file `credentials-velero` in the local machine.
++
+Please refer to https://github.com/vmware-tanzu/velero-plugin-for-gcp/tree/v1.0.0[Velero Plugin For GCP].
+
+... Azure Blob Storage
++
+Create the credential file `credentials-velero` in the local machine.
++
+Please refer to https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/tree/v1.0.0[Velero Plugin For Azure].
+
+.. S3-compatible Storage Providers
++
+Create the credential file `credentials-velero` in the local machine.
++
+```
+[default]
+aws_access_key_id=<S3_COMPATIBLE_STORAGE_ACCESS_KEY_ID>
+aws_secret_access_key=<S3_COMPATIBLE_STORAGE_SECRET_ACCESS_KEY>
+```
++
+
+. Velero CLI
++
+[source,bash]
+----
+sudo zypper install velero
+----
+
+== Limitations
```

Comment:
Also have a reference link to the Velero documentation for the detailed known issues and limitations.